samples/authorization/csm_authorization_proxy_server_v200-alpha.yaml

```diff
@@ -107,5 +107,6 @@ metadata:
 data:
   csm-config-params.yaml: |-
     CONCURRENT_POWERFLEX_REQUESTS: 10
+    CONCURRENT_POWERSCALE_REQUESTS: 10
```

Comment:
should this be going into alpha version?

Reply:
No, fixed.